project/cmake/CMakeLists.txt

```diff
@@ -450,6 +450,14 @@ configure_file(${PROJECT_SOURCE_DIR}/scripts/common/AddonHelpers.cmake
 configure_file(${PROJECT_SOURCE_DIR}/scripts/common/AddOptions.cmake
                ${CORE_BUILD_DIR}/lib/${APP_NAME_LC}/AddOptions.cmake COPYONLY)
 
+# Documentation
+find_package(Doxygen REQUIRED)
```

Comment:
please don't make this required. Doxygen with all its deps is huge. Many users/packagers don't need the docs, so no need to force this dep on them.